android/src/main/java/com/getcapacitor/community/audio/AudioDispatcher.java

```diff
@@ -92,6 +92,7 @@ public void setVolume(float volume) throws Exception {
 
   public void loop() throws Exception {
     mediaPlayer.setLooping(true);
+    invokePlay(0.0, true);
```

Comment:
Why do we need to play from scratch when we want to loop the current music

Reply:
Can you reply on the last comment?
It addresses your question as well

Reply:
@haimomesi I have work tasks where I need to use `loop` without `play`. These are different methods, you should separate them and let the app call them when necessary